ui/app/AppLayouts/Wallet/popups/buy/BuyCryptoModal.qml

```diff
@@ -88,7 +88,7 @@ StatusStackModal {
         readonly property ModelEntry selectedTokenEntry: ModelEntry {
             sourceModel: root.plainTokensBySymbolModel
             key: "key"
-            value: root.buyCryptoInputParamsForm.selectedTokenKey
+            value: root.buyCryptoInputParamsForm.selectedTokenKey || Constants.ethToken
```

Comment:
... here, so we can see whether we explicitly requested to buy some specific token, or just any token